specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2020-09-01-preview/examples/SqlMiCreateOrUpdateDatabaseMigrationMIN.json

```diff
@@ -0,0 +1,82 @@
+{
+  "parameters": {
+    "scope": "subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/managedInstance1",
+    "targetDbName": "db1",
+    "api-version": "2020-09-01-preview",
+    "parameters": {
+      "location": "northeurope",
+      "properties": {
+        "sourceDatabaseName": "aaa",
+        "scope": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/instance",
```

Comment:
>scope [](http://example.com/codeflow?start=9&length=5)

Is this different from the scope parameter?

This should be returned in the response. (all properties in the put should be returned except secrets)

Reply:
Why scope needs to be specified both in the URL (line 3) and the body (line 10)? Are they different?

If the scope in the body is needed in the request body, the response also should return it.